[48.982858658] 2018-04-15 17:15:48,742 - INFO - launcher: Starting sanity checks
[48.982858658] 2018-04-15 17:15:48,742 - INFO - launcher: Checking for Python 3.5+
[49.966573715] 2018-04-15 17:15:48,743 - INFO - launcher: Checking console encoding
[49.966573715] 2018-04-15 17:15:48,743 - INFO - launcher: Setting console encoding to UTF-8
[49.966573715] 2018-04-15 17:15:48,743 - INFO - launcher: Ensuring we're in the right environment
[53.001642227] 2018-04-15 17:15:48,746 - INFO - launcher: Adding local bins/ folder to path
[55.004358292] 2018-04-15 17:15:48,748 - INFO - launcher: Required checks passed.
[70.977210999] 2018-04-15 17:15:48,764 - INFO - launcher: Optional checks passed.
[71.943521500] 2018-04-15 17:15:48,765 - INFO - launcher: Moving old musicbot log

######################### PRE-RUN SANITY CHECKS PASSED #########################

[69.6023976802825928] 2018-04-15 17:16:58,295 - WARNING - musicbot.config | In config.py::MainThread(14680), line 112 in run_checks: i18n file does not exist. Trying to fallback to config/i18n/en.json.
[69.8719966411590576] 2018-04-15 17:16:58,565 - INFO - musicbot.config | In config.py::MainThread(14680), line 123 in run_checks: Using i18n: config/i18n/en.json
[69.9974358081817627] 2018-04-15 17:16:58,690 - DEBUG - musicbot.json | In json.py::MainThread(14680), line 8 in __init__: Init JSON obj with config/i18n/en.json
[72.9379718303680420] 2018-04-15 17:17:01,631 - DEBUG - musicbot.bot | In bot.py::MainThread(14680), line 227 in _setup_logging: Set logging level to INFO
[72.9379718303680420] 2018-04-15 17:17:01,631 - INFO - musicbot.bot | In bot.py::MainThread(14680), line 84 in __init__: Starting MusicBot 1.9.7
[72.9389722347259521] 2018-04-15 17:17:01,632 - INFO - musicbot.bot | In bot.py::MainThread(14680), line 90 in __init__: Loaded autoplaylist with 2531 entries
[82.6156692504882812] 2018-04-15 17:17:11,309 - DEBUG - musicbot.bot | In bot.py::MainThread(14680), line 1074 in on_ready: Connection established, ready to go.
[82.6166160106658936] 2018-04-15 17:17:11,310 - DEBUG - musicbot.bot | In bot.py::MainThread(14680), line 339 in _cache_app_info: Caching app info
[82.9549674987792969] 2018-04-15 17:17:11,648 - DEBUG - musicbot.bot | In bot.py::MainThread(14680), line 889 in _scheck_ensure_env: Ensuring data folders exist
[82.9570853710174561] 2018-04-15 17:17:11,650 - DEBUG - musicbot.bot | In bot.py::MainThread(14680), line 905 in _scheck_server_permissions: Checking server permissions
[82.9570853710174561] 2018-04-15 17:17:11,650 - DEBUG - musicbot.bot | In bot.py::MainThread(14680), line 909 in _scheck_autoplaylist: Auditing autoplaylist
[82.9570853710174561] 2018-04-15 17:17:11,650 - DEBUG - musicbot.bot | In bot.py::MainThread(14680), line 913 in _scheck_configs: Validating config
[82.9579679965972900] 2018-04-15 17:17:11,651 - DEBUG - musicbot.config | In config.py::MainThread(14680), line 212 in async_validate: Validating options...
[82.9579679965972900] 2018-04-15 17:17:11,651 - DEBUG - musicbot.bot | In bot.py::MainThread(14680), line 916 in _scheck_configs: Validating permissions config
[82.9579679965972900] 2018-04-15 17:17:11,651 - DEBUG - musicbot.permissions | In permissions.py::MainThread(14680), line 64 in async_validate: Validating permissions...
[82.9579679965972900] 2018-04-15 17:17:11,651 - INFO - musicbot.bot | In bot.py::MainThread(14680), line 1091 in on_ready: Connected: 434907824640753674/๖ۣۜZΞUS⚡#1169
[82.9589655399322510] 2018-04-15 17:17:11,652 - INFO - musicbot.bot | In bot.py::MainThread(14680), line 1099 in on_ready: Owner:     278136676574953472/๖ۣۜZΞUS⚡#4286

[82.9589655399322510] 2018-04-15 17:17:11,652 - INFO - musicbot.bot | In bot.py::MainThread(14680), line 1102 in on_ready: Server List:
[82.9599351882934570] 2018-04-15 17:17:11,653 - INFO - musicbot.bot | In bot.py::MainThread(14680), line 1105 in on_ready:  - Fury's Basement
[82.9599351882934570] 2018-04-15 17:17:11,653 - INFO - musicbot.bot | In bot.py::MainThread(14680), line 1105 in on_ready:  - Official ๖ۣۜZΞUS⚡Chat
[82.9609653949737549] 2018-04-15 17:17:11,654 - INFO - musicbot.bot | In bot.py::MainThread(14680), line 1151 in on_ready: Not bound to any text channels
[82.9609653949737549] 2018-04-15 17:17:11,654 - INFO - musicbot.bot | In bot.py::MainThread(14680), line 1177 in on_ready: Not autojoining any voice channels
